config_defaults/subtests/docker_cli/systemd.ini

```diff
@@ -0,0 +1,20 @@
+[docker_cli/systemd]
+#: Systemd unit file ``sysd_unit_file``
+sysd_unit_file = p4321.service
+#: Systemd unit file destination ``sysd_unitf_dest``
+sysd_unitf_dest = /etc/systemd/system/p4321.service
+
+#: increase ``docker_timeout`` to 180 as building image can take longer
+docker_timeout = 300
+
+#: build image ``name``
+name=p4321
+
+#: image building args ``build_options_csv``
+build_options_csv = --force-rm,-t,%(name)s
+
+#: local ip address ``tcp_ip``
+tcp_ip = 127.0.0.1
+
+#: port to use ``tcp_port``
```

Comment:
Anyhow, all these comments should probably be filled in a little better with descriptions of what the fields are for.